.github/workflows/sass.yaml

```diff
@@ -45,6 +45,14 @@ jobs:
         run: |
           ! grep "\$govuk-" .tmp/all.css
 
+      - name: Save compiled Sass
+        uses: actions/upload-artifact@v4.3.1
+        if: ${{ !cancelled() }}
+        with:
+          name: Dart Sass v1.0.0 output
```

Comment:
**nitpick** Would have wanted to re-use the name of the job by looking in one of the [contexts](https://docs.github.com/en/actions/learn-github-actions/contexts) provided to the step... but couldn't find anything. If anyone knows how, would be keen to update. 

Don't think that's a blocker, though, more of a nice-to-have-if-we-know-how.

Reply:
Uploading the artefact is also very repetitive – we briefly explored running these using a matrix in #3030 but at the time it felt like it made the code too complicated. Wondering if that's worth revisiting.

Equally with us dropping support for Ruby Sass and LibSass 'soon' maybe it's not worth spending any more time on this? 😕

Reply:
Yeah, thought there was a fair amount of repetitions and remembered Colin's attempt at having a matrix. Still not convinced we need to DRY things up with a matrix there given it's not that costly to add something 6 times and we don't touch those that often. Especially with the change of Sass compiler support at some point in the future.

If for whatever reason we start bulking these workflows with more shared steps, we could revisit then.